static/app/views/onboarding/onboarding.tsx

```diff
@@ -22,19 +21,6 @@ import SdkConfiguration from './sdkConfiguration';
 import {StepData, StepDescriptor} from './types';
 import OnboardingWelcome from './welcome';
 
-type AnalyticsOpts = {
-  organization: Organization;
-  project: Project | null;
-  step: StepDescriptor;
-};
-
-const recordAnalyticStepComplete = ({organization, project, step}: AnalyticsOpts) =>
```

Comment:
@roggenkemper wondering why we are removing this event altogether instead of replacing it